src/components/table/examples/table-sortable/index.njk

```diff
@@ -63,6 +66,18 @@
                         {
                             "value": "<span class='ons-status ons-status--success'>Ready</span>",
                             "dataSort": "0"
+                        },
+                        {
+                            "form": {
+                                "method": "Get",
+                                "action": "#" 
+                                "button": {
+                                    "type": "button",
+                                    "text": "View details",
+                                    "classes": "ons-btn--secondary",
```

Comment:
If you look at the button component macro options you can set the button style using `variants: 'secondary'` instead of using the `classes` property

Reply:
I'll try to use your other suggestion see what happens

Reply:
Removed classes in the form options and added another way of adding a button via value option